focuspoints.lrdevplugin/FujifilmDelegates.lua

```diff
@@ -0,0 +1,92 @@
+--[[
+  Copyright 2016 Joshua Musselwhite, Whizzbang Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--]]
+
+--[[
+  A collection of delegate functions to be passed into the DefaultPointRenderer when
+  the camera is Fuji
+--]]
+
+local LrStringUtils = import "LrStringUtils"
+require "Utils"
+
+FujifilmDelegates = {}
+
+--[[
+-- metaData - the metadata as read by exiftool
+--]]
+function FujifilmDelegates.getAfPoints(photo, metaData)
+  local focusPoint = ExifUtils.findFirstMatchingValue(metaData, { "Focus Pixel" })
+  if focusPoint == nil then
+    return nil
+  end
+  local values = split(focusPoint, " ")
+  local x = LrStringUtils.trimWhitespace(values[1])
+  local y = LrStringUtils.trimWhitespace(values[2])
+  if x == nil or y == nil then
+    return nil
+  end
+
+  local imageWidth = ExifUtils.findFirstMatchingValue(metaData, { "Exif Image Width" })
+  local imageHeight = ExifUtils.findFirstMatchingValue(metaData, { "Exif Image Height" })
+  if imageWidth == nil or imageHeight == nil then
+    return nil
+  end
+
+  local orgPhotoWidth, orgPhotoHeight = parseDimens(photo:getFormattedMetadata("dimensions"))
+  local xScale = orgPhotoWidth / imageWidth
+  local yScale = orgPhotoHeight / imageHeight
+  if orgPhotoWidth < orgPhotoHeight then
+    xScale = orgPhotoHeight / imageWidth
+    yScale = orgPhotoWidth / imageHeight
+  end
+
+  local result = {
+    pointTemplates = DefaultDelegates.pointTemplates,
+    points = {
+      {
+        pointType = "af_selected_focus",
```

Comment:
what does this magic string do? Is it used or read somewhere? Is so, perhaps some type of constant so we can avoid spelling mistakes if something gets refactored.